packages/gluestick/src/cli/index.js

```diff
@@ -212,4 +200,4 @@ commander.command('*', null, { noHelp: true }).action(cmd => {
   commander.help();
 });
 
-commander.parse(process.argv);
+exports.default = commander;
```

Comment:
You don't have to export it with `default` - `module.exports = commander` is fine.